docs/best-practices/contributing/readme.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to repositories
+
+This section summarizes best practices for contributing to code repositories.
+
+*TL;DR: Reach out to the repository owners before making a pull-request, keep contributions focussed.*
+
+### Discuss the scope of your contribution
+Whether contributing to a repository owned by your team, or to another team's product or library, the first step should
+almost always (see below for exceptions) be to reach out to the team and discuss your contribution:
+
+- Check the relevant GitHub or JIRA issues for related work
```

Comment:
probably also want to look for a contributions file that describes specializations for a particular project